app/presenters/search_presenters.rb

```diff
@@ -124,10 +124,12 @@ def item
     end
 
     def options
-      super.merge(:classes => [Topic, CMS::Page])
+      super.merge(:classes => [CMS::Page])
```

Comment:
```suggestion
      super.merge(classes: [CMS::Page])
```

as we are changing this line anyway...